features/projects/connections.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Projects should show links to the connected APIs
+  As a User
+  So that I can easily navigate to Project apps
+  I want to see all connected app links on the show page
+
+  Background:
+    Given the following users exist
+      | first_name | last_name | email            | receive_mailings  |
+      | Bill       | Bob       | Bill@example.com | true              |
+
+    Given the following projects exist:
+      | title   | description | status   | author | pivotaltracker_url                               | github_url               | slack_channel_name |
+      | hello   | earthlings  | active   | Bill   | https://www.pivotaltracker.com/n/projects/742821 | https://github.com/hello | hello_earthlings   |
+      | Bat Man | All bat     | active   | Bill   |                                                  |                          |                    |
+
+  Scenario: I can see a link to the GitHub project page
```

Comment:
I wonder if some of this could be condensed down with a "Scenario Outline"?

Reply:
Good point. I will take a look at that!

Reply:
feel free to open a new refactoring ticket for that :-)